### PR TITLE
add styling

### PR DESCRIPTION
**Pull request by sulaimonaa**
- Add styling to the UI of the app
- Increase the font size of the completed percentage status
- Increased the size progress status bar
- Use the required colors as on the big picture

### CODE REVIEW

**Review by aamirkhan2478 — COMMENTED** [review]
Hi @sulaimonaa, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

You have submitted a project, that hasn't a descriptive and meaningful PR summary kindly add a descriptive and meaningful PR summary and also please add good and descriptive commit messages as well for the next push :slightly_smiling_face: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/styles.module.css [hunk 1]
```diff
@@ -161,7 +161,11 @@ form input {
...
 .progress_btn {
-  width: 40px;
+  width: 60px;
+}
+
+.completed_stat {
+  font-size: 34px;
 }
 
 .current_chapter_title {
```
Comment (nahnahdev) [inline src/styles.module.css:196]: - [ ] Great job so far @sulaimonaa There are just a few changes I will suggest you implement and you'll be on your way to the  next Milestone :+1: 
- [ ] I have mentioned some style fixes that need to be made in the screenshot below. So kindly fix them so your design looks exactly like the Given design :wink: 
- Your Design
image

- Zeplin Design
image

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @sulaimonaa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :fire: 
- Linters well setup and passing :heavy_check_mark: 
- Good use of GitFlow :heavy_check_mark: 
- CSS good practices used :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @sulaimonaa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] Application works as expected :white_check_mark: 
- [x] Pull request has a descriptive title and summary :white_check_mark: 
- [x] Implemented all the changes from the previous reviewer :white_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**